### PR TITLE
Improve an expression from 'value's to 'values' for taint-and-toleration

### PR DESCRIPTION
**Pull request by haripriya9647**
Issue - #44209

### CODE REVIEW

File: content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ The default value for `operator` is `Equal`.
 A toleration "matches" a taint if the keys are the same and the effects are the same, and:
 
 * the `operator` is `Exists` (in which case no `value` should be specified), or
-* the `operator` is `Equal` and the `value`s are equal.
+* the `operator` is `Equal` and the `values` are equal.
```
Comment (windsonsea) [inline content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md:74]:
```suggestion
* the `operator` is `Equal` and the values should be equal.
```
How about using a more descriptive expression for this?

Comment (haripriya9647) [reply content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md]: please check my latest commit

Comment (windsonsea) [reply content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md]: lgtm, thanks.

**Comment by windsonsea** [comment]
You may need to squash your commits into a single one for this small change.

**Comment by haripriya9647** [comment]
Sure I will do it

**Comment by haripriya9647** [comment]
Done @windsonsea please review

**Comment by windsonsea** [comment]
/retitle Improve an expression from 'value's to 'values' for taint-and-toleration
/lgtm

**Pull request merged**